src/Rules/Documentation.hs

```diff
@@ -10,6 +10,81 @@ import Settings
 import Target
 import Utilities
 
+docPackage :: Package
+docPackage = hsLibrary "Documentation" "docs"
+
+-- | Build all documentation
+documentationRules :: Rules ()
+documentationRules = do
+    buildHtmlDocumentation
+    buildPdfDocumentation
+    buildDocumentationArchives
+    "docs" ~> do
+        root <- buildRoot
+        -- Archives `need` the associated html
+        let archives = map pathArchive docPaths
+            pdfs = map pathPdf $ drop 1 docPaths
```

Comment:
Perhaps, `drop 1 docPaths` could be replaced with `docPaths \\ [ "libraries" ]` to avoid relying on the order and make it a bit more readable?

Alternatively, `pdfs = map pathPdf ["users_guide", "Haddock"]` doesn't look too bad either.